rotors_control/src/nodes/lee_position_controller_node.cpp

```diff
@@ -123,24 +107,22 @@ void LeePositionControllerNode::CommandMultiDofJointTrajectoryCallback(
     return;
   }
 
-  mav_msgs::EigenCommandTrajectoryPositionYaw eigen_reference;
-  mav_msgs::eigenCommandTrajectoryPositionYawFromMultiDofJointTrajectoryPointMsg(msg->points.front(),
-                                                                                 &eigen_reference);
+  mav_msgs::EigenTrajectoryPoint eigen_reference;
+  mav_msgs::eigenTrajectoryPointFromMsg(msg->points.front(), &eigen_reference);
```

Comment:
The only way this can fail is if the trajectory is empty, though, which is easy to check by checking length of transform (and perhaps an empty one is still a valid one anyway)? Do you think this is really necessary?